employee_seniority_level/models/account_analytic_line.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class AccountAnalyticLine(models.Model):
+
+    _inherit = 'account.analytic.line'
+
+    @api.model
+    def create(self, vals):
```

Comment:
can we split this to smaller methods pls?

Reply:
It has been split